spec/api-browser-window-spec.ts

```diff
@@ -5684,17 +5684,23 @@ describe('BrowserWindow module', () => {
     ifdescribe(process.platform !== 'darwin')('when fullscreen state is changed', () => {
       it('correctly remembers state prior to fullscreen change', async () => {
         const w = new BrowserWindow({ show: false });
+
+        // This should do nothing.
```

Comment:
_(nit)_ this comment and the ones below are clear from the context in this PR, but if I came across it cold when reading the `.ts` file I would find it a little confusing.

A little wordier, but maybe `// This should be a no-op since this is already the window's current state`

Reply:
Sounds good to me.

```suggestion
        // This should be a no-op since this is already the window's current state
```

Edit: whoops, there is more such comments.